classes/parser.php

```diff
@@ -674,4 +674,4 @@ public function getModificationDate()
     }
 }
 
-?>
\ No newline at end of file
+?>
```

Comment:
Hello,

Please remove the newline at the end of this file.

Cheers,
Brookins Consulting